config/locales/en/course/discussion/post.yml

```diff
@@ -0,0 +1,5 @@
+en:
+  course:
+    discussion:
+      post:
+        confirm-delete-post: 'Are you sure that you want to delete this post?'
```

Comment:
@fonglh it's a valid concern, but in the context of viewing this in as a comment in a mission, `delete this` might be a little ambiguous. 

If you had to force a suggestion out of me, I will suggest `Are you sure you want to delete this comment/reply?`.